src/main/java/com/eternalcode/discordapp/DiscordApp.java

```diff
@@ -61,6 +65,12 @@ public static void main(String... args) throws InterruptedException {
         configManager.load(databaseConfig);
         configManager.load(experienceConfig);
 
+        ConfigManager dataManager = new ConfigManager(new File("data"));
+        UserOnVoiceChannel userOnVoiceChannel = new UserOnVoiceChannel();
+        dataManager.load(userOnVoiceChannel);
+
+        userOnVoiceChannel.addUserOnVoiceChannel(0L, Instant.now());
```

Comment:
Po co?

Reply:
Ze względu na zastosowanie hashMap'y muszę dodać jakąś jedną wartość podczas tworzenia pliku z danymi dlatego zostało to tutaj umieszczone

Reply:
W jakim celu?

Reply:
W celu takim iż gdy początkowo bot wygeneruje plik z pustą listę przy ponownym uruchomieniu bota wyrzuca nam exception przy próbie przetworzenia Collections#emptyMap na HashMapę

Reply:
Czemu migrujesz Kolekcje na hashmape? gdzie to robisz?

Reply:
Czemu migrujesz Kolekcje na hashmape? gdzie to robisz?

Reply:
Właśnie ja tego nie migruję ponieważ ja nie tworzę kolekcji - problem występuje gdy jak widać 
```
 public HashMap<Long, Long> usersOnVoiceChannel = new HashMap<>();
 ```
 
utworzy sobie pustą mapę i próbuje ją bot załadować po ponownym uruchomieniu

Reply:
> Właśnie ja tego nie migruję ponieważ ja nie tworzę kolekcji - problem występuje gdy jak widać
> 
> ```
>  public HashMap<Long, Long> usersOnVoiceChannel = new HashMap<>();
> ```
> 
> utworzy sobie pustą mapę i próbuje ją bot załadować po ponownym uruchomieniu

To jak w końcu jest? "przy próbie przetworzenia Collections#emptyMap na HashMapę" "Właśnie ja tego nie migruję"

Reply:
> > Właśnie ja tego nie migruję ponieważ ja nie tworzę kolekcji - problem występuje gdy jak widać
> > ```
> >  public HashMap<Long, Long> usersOnVoiceChannel = new HashMap<>();
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > utworzy sobie pustą mapę i próbuje ją bot załadować po ponownym uruchomieniu
> 
> To jak w końcu jest? "przy próbie przetworzenia Collections#emptyMap na HashMapę" "Właśnie ja tego nie migruję"

Pokazywałem mój problem podczas streama na discordzie lecz tutaj go zobrazuję -

Bot przy pierwszym uruchomieniu tworzy sobię pustą mapę

![image](https://user-images.githubusercontent.com/60399586/233169067-d023f568-60a0-49de-b0dc-8d57ea4807be.png)

I przy następnym uruchomieniu throwuje exception jeżeli mapa jest pusta

```Exception in thread "main" panda.std.AttemptFailedException: java.lang.IllegalArgumentException: Can not set java.util.HashMap field com.eternalcode.discordapp.experience.data.UsersVoiceActivityData.usersOnVoiceChannel to java.util.Collections$EmptyMap
```

Reply:
Immutable mapką chyba można to rozwiązać 
```java
 public HashMap<Long, Long> usersOnVoiceChannel = new ImmutableMap...
 ```

Reply:
> Immutable mapką chyba można to rozwiązać
> 
> ```java
>  public HashMap<Long, Long> usersOnVoiceChannel = new ImmutableMap...
> ```

Tak, lecz ja tutaj potrzebuję by mapa była modyfikowalna. Kiedy zastosuję `new ImmutableMap..` wtedy metody HashMap#put etc. nie będą działać

Reply:
To zwykla map